library/open-liberty

```diff
@@ -15,6 +15,15 @@ Directory: releases/latest/full
 Architectures: amd64, ppc64le, s390x
 File: Dockerfile.ubuntu.adoptopenjdk8
 
+Tags: 20.0.0.7-kernel-java8-openj9
+Directory: releases/20.0.0.7/kernel
+Architectures: amd64, ppc64le, s390x
+File: Dockerfile.ubuntu.adoptopenjdk8
+
+Tags: 20.0.0.7-full-java8-openj9
+Directory: releases/20.0.0.7/full
+Architectures: amd64, ppc64le, s390x
```

Comment:
Ah! Looks like the section is missing a `File: Dockerfile.ubuntu.adoptopenjdk8`?